applications/xe-cnf/helm/exe-health/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```

Comment:
This file doesn't have Copyright header. This is helm configuration file. What is the need to have this file in repo. Can you please remove this file?